app/src/main/kotlin/com/wire/android/ui/main/MainScreen.kt

```diff
@@ -0,0 +1,57 @@
+package com.wire.android.ui.main
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavBackStackEntry
+import com.wire.android.R
+import com.wire.android.ui.main.archive.ArchiveScreen
+import com.wire.android.ui.main.convesations.ConversationsScreen
+import com.wire.android.ui.main.settings.SettingsScreen
+import com.wire.android.ui.main.support.SupportScreen
+import com.wire.android.ui.main.vault.VaultScreen
+
+enum class MainScreen(
```

Comment:
Maybe we could rename it to improve descriptiveness?
```suggestion
enum class MainNavigationScreenItem(
```